ui/app/AppLayouts/Wallet/stores/TokensStore.qml

```diff
@@ -56,8 +66,8 @@ QtObject {
                 expression: tokenIcon(model.symbol)
             },
             ExpressionRole {
-                name: "jsArraySources"
-                expression: model.sources.split(";")
+                name: "networkChainName"
+                expression: model.chainName + (model.isTest? " " + qsTr("(Test)") : "")
             }
```

Comment:
I think it's too early to do that transformation. UI should be free to handle test networks in any way (by adding that "Test" suffix or any other way - e.g. by special color or icon...). So view should use `isTest` directly and then we don't need this expression role here.

Reply:
I agree